<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWget_libros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaInfo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Codigo</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Titulo</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Descrip.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Autor</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Categoria</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Editorial</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Informacion</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
